action.yml

```diff
@@ -8,6 +8,9 @@ inputs:
   INSPECTIONS_FILE:
     description: Path to inspections file relative to root of project directory
     required: True
+  IGNORE_FILE_PATTERN:
+    description: Regular expression for the files to be ignored.
```

Comment:
Please specify details on exactly how to pass in the arguments. Using JSON in YAML is not common and is unexpected.

Reply:
```
A list of regular expressions for files whose inspection results are to be ignored.
The inputs are specified within square brackets where each input regular expression is enclosed within double quotes, comma separated and with correct indentations.
```
Is this okay?